### ISSUE TITLE
not linked with lm

### ISSUE THREAD

**Issue by kostya** [jessedoyle/duktape.cr#1]
i try to compile example from readme

```
duktape.c:(.text+0xb84): undefined reference to `ceil'
/home/kostya/projects/1112/libs/duktape/.build/lib/libduktape.a(libduktape.o): In function `duk__atan':
duktape.c:(.text+0xb89): undefined reference to `atan'
/home/kostya/projects/1112/libs/duktape/.build/lib/libduktape.a(libduktape.o): In function `duk__asin':
duktape.c:(.text+0xb8e): undefined reference to `asin'
/home/kostya/projects/1112/libs/duktape/.build/lib/libduktape.a(libduktape.o): In function `duk__acos':
duktape.c:(.text+0xb93): undefined reference to `acos'
/home/kostya/projects/1112/libs/duktape/.build/lib/libduktape.a(libduktape.o): In function `duk__round_fixed':
duktape.c:(.text+0x19bd): undefined reference to `floor'
/home/kostya/projects/1112/libs/duktape/.build/lib/libduktape.a(libduktape.o): In function `duk__fmin_fixed':
duktape.c:(.text+0x1a32): undefined reference to `fmin'
/home/kostya/projects/1112/libs/duktape/.build/lib/libduktape.a(libduktape.o): In function `duk__fmax_fixed':
duktape.c:(.text+0x1aa4): undefined reference to `fmax'
collect2: error: ld returned 1 exit status
Error: execution of command failed with code: 1: `cc -o 1 "/home/kostya/projects/1112/.crystal/home/kostya/projects/1112/1.cr/main.o"   -L/home/kostya/projects/1112/libs/duktape/.build/lib -L/home/kostya/projects/1112/libs/duktape/.build/include -lduktape -levent -lrt -lpcl -lpcre -lgc -lpthread -lunwind`

```

if i add -lm after -lduktape all is compiled

**Comment by jessedoyle** [comment]
Thanks for the feedback!

You're right, all of the Duktape documentation says to link with the standard math library, but clang on my system would throw a warning about unused compiler arguments.

Please let me know if pull request #2 fixes the issue.

**Comment by kostya** [comment]
#2 not fixed
crystal failed to link

```
`cc -o 1 "./.crystal/home/kostya/projects/1113/1.cr/main.o"   -L/home/kostya/projects/1113/libs/duktape/.build/lib -L/home/kostya/projects/1113/libs/duktape/.build/include -lduktape -levent -lrt -lpcl -lpcre -lgc -lpthread -lunwind`
```

if add -lm

```
`cc -o 1 "./.crystal/home/kostya/projects/1113/1.cr/main.o"   -L/home/kostya/projects/1113/libs/duktape/.build/lib -L/home/kostya/projects/1113/libs/duktape/.build/include -lduktape -levent -lrt -lpcl -lpcre -lgc -lpthread -lunwind -lm` 
```

all is linked

**Comment by jessedoyle** [comment]
@kostya  - I think I understand.

I've updated PR #2 to link against the standard math library in Crystal's `ldflags`. I believe this should solve your problem.

Can you verify that your issue is now resolved with PR #2?

Thanks!

**Comment by kostya** [comment]
fixed now
